examples/k8s/hz-go-service/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.17
 
 require (
 	github.com/gorilla/mux v1.8.0
-	github.com/hazelcast/hazelcast-go-client v1.1.2-0.20220131082557-ec272b3372a8
+	github.com/hazelcast/hazelcast-go-client v1.2.0
```

Comment:
Could you use v1.3.0?

Reply:
Updated, [9c4baec](https://github.com/hazelcast/hazelcast-go-client/pull/743/commits/9c4baecc96452e673ce3249504f10ce2efa3d810).